website/static/js/registrationUtils.js

```diff
@@ -153,6 +185,17 @@ var Question = function(data, id) {
             return new Comment(comment);
         })
     );
+	self.deleteComment = function(comment) {
+		self.comments.remove(comment);
+		comment.isDeleted = true;
+
+		var commentList = document.getElementById('commentList');
```

Comment:
Instead of this, I would rather see:

```javascript
// in the Comment constructor
self.isDeleted = ko.observable(false);
self.isDeleted.subscribe(function(isDeleted) {
  if (isDeleted) {
    self.value('this comment was deleted');
  }
});
```